[doc] Organization admin (#67)
OpenCTI-Platform/docs: docs/administration/users.md

## Organizations

Users can belong to organizations, which is an additional layer of [data segregation](segregation.md) and customization.

## Organization administration

> 

Reviewer: ```suggestion
!!! tip "Enterprise edition"

    Playbooks automation is available under the "Filigran Entreprise Edition" license.

    [Please read the dedicated page to have all information](../enterprise.md)
```

Plateform administrators can promote members of an organization as "Organization administrator". This elevated role grants them the necessary capabilities to create, edit and delete users from the corresponding Organization. Additionally, administrators have the flexibility to define a list of groups that can be granted to newly created members by the organization administrators. This feature simplifies the process of granting appropriate access and privileges to individuals joining the organization.

> 

Reviewer: ```suggestion
For example, Company A uses OpenCTI for all its infosecurity teams. Each infosecurity teams are situated in different Business Units. Platform administrators of Company A have created an "Organization" for each Business Unit and members of infosecurity teams are part of their respective Organization. But Company A is a big company, platform administrators cannot handle the management of user account related to Business Units infosec team. So, they have promoted on member of each infosec teams as an "Organization administrator". Each of them can create accounts for new members of their team and grant them Groups related to their teams. But, as designed by Platform administrators, they will not be able to grant new members with the infamous "Central access" Group that define access for Platform administrators.
```

![Organization admin Settings view](assets/organization_admin_view.png)

> The platform administrator can promote/demote an organization admin through its user edition form.

Reviewer: ```suggestion
A Platform Administrator can promote/demote an "Organization administrator" through its user edition form.
```

![Organization admin promoting/demoting](assets/define_organization_admin.png)

> The "Organization admin" has restricted access to Settings. They can only manage the members of the organizations for which they have been promoted as "admins".

Reviewer: ```suggestion
"Organization administrators" have restricted access to Settings. They can only manage users that are part of Organizations for which they have been promoted as "Organization administrators".
```
